content/pl/concepts/overview/components.md

```diff
@@ -0,0 +1,117 @@
+---
+reviewers:
+- lavalamp
```

Comment:
Can you remove these lines? We want to use the reviewers/approvers from the OWNERS file instead. Only when the localization will be having a lot of people, files can be appointed to specific contributors.